### DISCUSSION TITLE
Issues uninstalling

### DISCUSSION THREAD

**Discussion by kyle-litton** [chris1111/Wireless-USB-Big-Sur-Adapter#40] [Q&A]
I ended up using a PCIE Wifi card that is supported. I am having trouble uninstalling, I removed the kexts and rebooted however the icon is still in the menu bar. I am not sure where the install put the files that I would need to delete.

**Comment by chris1111** [comment]
Select only Uninstall

**Reply by kyle-litton** [reply]
Ah I knew it’d be something obvious! Thanks for the help and all the work put into this project!
